modules/kuberay-cluster/kuberay-values.yaml

```diff
@@ -236,85 +236,14 @@ worker:
         name: ray-logs
```

Comment:
same, wipe all references to T4 if we're updating cluster creation instructions to not include T4 nodepool. if you want, make this part a different PR so it's well tested. we need to ensure the ray workers come up on both standard and AP on L4s.

Reply:
Leaving this as-is and reverting README changes to Standard mode. I have not tested this on GKE Standard.